package.json

```diff
@@ -13,12 +13,12 @@
   "dependencies": {
     "@reduxjs/toolkit": "^1.6.1",
     "plotly.js-dist": "^2.2.0",
-    "react-grid-layout": "^1.2.5",
+    "react-grid-layout": "^0.16.2",
```

Comment:
you can remove this and the `@types/react-grid-layout` if it's the same version defined in Dashboards' package.json

Reply:
Removed 👍🏽